WordPress/gutenberg
# Update the update icon

## What?
In https://github.com/WordPress/gutenberg/pull/46438 I attempted to update the SVG for the update icon but somehow those changes were lost, so this is another attempt.

## Testing Instructions
0. Enable the off canvas editing experiment for the navigation block
1. Add a navigation block
2. Add a page list to the navigation 
3. Check that the icon looks like this:
<img width="255" alt="Screenshot 2022-12-15 at 09 20 52" src="https://user-images.githubusercontent.com/275961/207821310-04d86ba0-a9bc-49f9-b950-a5c0a22bece4.png">


### Testing Instructions for Keyboard
These are only visual changes
